tools/package-release/src/commands/publish/index.ts

```diff
@@ -111,17 +119,20 @@ export default class PackageRelease extends Command {
 			CliUx.ux.action.start( `${ verb } ${ name }` );
 
 			try {
-				const cwd = getFilepathFromPackageName( name );
-				return execSync(
-					`SKIP_TURBO=true pnpm publish ${
-						dryRun ? '--dry-run' : ''
-					} --publish-branch=${ branch }`,
-					{
-						cwd,
-						encoding: 'utf-8',
-						stdio: 'inherit',
-					}
-				);
+				if ( isValidUpdate( name ) ) {
+					const cwd = getFilepathFromPackageName( name );
+					return execSync(
+						`SKIP_TURBO=true pnpm publish ${
+							dryRun ? '--dry-run' : ''
+						} --publish-branch=${ branch }`,
+						{
+							cwd,
+							encoding: 'utf-8',
+							stdio: 'inherit',
+						}
+					);
+				}
+				this.log( `${ name } does not have anything to update.` );
```

Comment:
I think rather than logging here you should pass a custom message to `action.stop`. This makes the output clearer.

e.g. 

```
CliUx.ux.action.stop(
					`${ name } does not have anything to update.`
				);
```

The way things are now the output is a bit confusing because it says "x does not having anything to update" and then "publishing package x ... done"

If you call stop with a message it looks like:

`Publishing @woocommerce/number... @woocommerce/number does not have anything to update.`